packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.page.tsx

```diff
@@ -40,6 +44,23 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     };
   }
 
+  function onSuccessRedirectHref({
+    userDepartment,
+    newUserRoles,
+    userId,
+  }: Awaited<ReturnType<typeof handleRequest>>) {
+    const ADMIN_ROLES = ['3', '4', '5'];
```

Comment:
variable naming nitpicking, but:

```suggestion
    const ADMIN_ROLE_IDS = ['3', '4', '5'];
```